.github/workflows/build-extensions.yml

```diff
@@ -253,10 +253,12 @@ jobs:
     env:
       output_dir: build/plugins/wasi_nn
       test_dir: build/test/plugins/wasi_nn
-      build_options_manylinux2014_x86_64: -DWASMEDGE_PLUGIN_WASI_NN_BACKEND=PyTorch -DWASMEDGE_PLUGIN_WASI_NN_BACKEND=TensorFlowLite
-      build_options_manylinux2014_aarch64: -DWASMEDGE_PLUGIN_WASI_NN_BACKEND=TensorFlowLite
-      tar_names_manylinux2014_x86_64: wasi_nn-pytorch wasi_nn-tensorflowlite
-      tar_names_manylinux2014_aarch64: wasi_nn-tensorflowlite
+      build_options_both: -DWASMEDGE_PLUGIN_WASI_NN_BACKEND=TensorFlowLite
```

Comment:
Using the name `manylinux2014_all` is better. We may support more architecture in the future, but `both` is inaccurate.